app/services/yelp_service.rb

```diff
@@ -0,0 +1,27 @@
+require 'yelp'
+class YelpService
+
+  attr_reader :client
+
+
+  def city_search(city)
+    client.search(city)
+  end
+
+  def box_search(box, params)
+    client.search_by_bounding_box(box, params)
+  end
+
+  def radius_search(coordinates, search_terms, radius = 1000, sort = 2)
+    coordinates_hash = { latitude: coordinates[0], longitude: coordinates[1] }
+    client.search_by_coordinates(coordinates_hash, term: search_terms, radius_filter: radius, sort: sort)
+  end
+  
+  def client
+    @client = Yelp::Client.new({ consumer_key: ENV['yelp_consumer_key'],
+                        consumer_secret: ENV['yelp_consumer_secret'],
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.